charts/flyte-agent/README.md

```diff
@@ -0,0 +1,39 @@
+# flyteagent
+
+![Version: v0.1.10](https://img.shields.io/badge/Version-v0.1.10-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+
+A Helm chart for Flyte agent
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| additionalContainers | list | `[]` | Appends additional containers to the deployment spec. May include template values. |
+| additionalVolumeMounts | list | `[]` | Appends additional volume mounts to the main container's spec. May include template values. |
+| additionalVolumes | list | `[]` | Appends additional volumes to the deployment spec. May include template values. |
+| affinity | object | `{}` | affinity for flyteagent deployment |
+| agentSecret.name | string | `""` | Specify name of K8s Secret. Leave it empty if you don't need this Secret |
+| agentSecret.secretData | object | `{"data":{"username":"User"}}` | Specify your Secret (with sensitive data) or pseudo-manifest (without sensitive data). See https://github.com/godaddy/kubernetes-external-secrets |
```

Comment:
This is the updated link: https://github.com/external-secrets/external-secrets

Reply:
sorry, forgot to remove it